.evergreen/generate_evergreen_tasks.js

```diff
@@ -366,6 +381,9 @@ OPERATING_SYSTEMS.forEach(
       if (clientEncryption) {
         expansions.CLIENT_ENCRYPTION = true;
       }
+      if (msvsVersion) {
+        expansions.MSVS_VERSION = msvsVersion;
+      }
```

Comment:
What is this used for? I see that `$MSVS_VERSION` is plumbed through to the invocation of the test script, but I don't see it actually used for any logic there

Reply:
It's used in `install-dependencies.sh` [here](https://github.com/mongodb/node-mongodb-native/pull/2634/files?file-filters%5B%5D=.js&file-filters%5B%5D=.opts&file-filters%5B%5D=.sh#diff-222fecb89c4b98404ba34c589acb1245806777c93bf48a5cc861727823726c37R82): `npm config set msvs_version ${MSVS_VERSION}`